docs/markdown_editor_viewer.md

```diff
@@ -84,4 +84,15 @@ This conversion is needed for rendering content in WYSIWYG after the first load
 
 ### [Image upload plugin](../contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/plugins/image-upload)
 
-TBD
+#### Adding/editing images
```

Comment:
I've decided not to go into detail with this plugin documentation as there are some major updates in progress (@micahscopes will be updating documentation as soon as he's done with it).